website/pages/en/index.js

```diff
@@ -164,7 +167,10 @@ class Index extends React.Component {
                   </h2>
                   <div>
                     <MarkdownBlock>
-                      Generics allow classes and methods to be parameterized (i.e., a type associated when a class is instantiated or a method is called in the same vein as statically typed languages like C# and Java).
+                      **[Generics](https://docs.hhvm.com/hack/generics/some-basics)** allow classes and methods to be parameterized to any set of types, all while preserving type safety, and supports both **_[generic covariance and contravariance](https://docs.hhvm.com/hack/generics/variance)_** on a type parameter.
+                    </MarkdownBlock>
+                    <MarkdownBlock>
+                      You can also create **_[Reified Generics](https://docs.hhvm.com/hack/generics/reified-generics)_**—Generics with type information accessible at runtime.
```

Comment:
Generics is a really big and powerful feature in Hack. Rather than enumerating a few things you can do with generics, perhaps we should say something more general?

> Hack supports a rich set of generic types, including type parameters, constraints, associated type constants and even reification.